docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -10,113 +10,139 @@ keywords: changelog, table, materialize
+    kafka_topic = 'changelog',
     partitions = 1,
-    value_format = 'avro'
+    value_format = 'JSON'
 );
 ```
 
-Insert some rows into `s1`, repeating entries for some of the keys.
+Now, you can view the latest values for each key in your `changelog` topic by issuing a pull query against the `latest_view` table that we just created above:
+```sql
+SELECT * FROM latest_view;
++------+------+------+------+
+|K     |V1    |V2    |V3    |
++------+------+------+------+
+|k1    |2     |c     |false |
+|k2    |4     |e     |true  |
+|k3    |3     |d     |true  |
+```
+
+Notice how for each key, the columns reflect the latest set of values.
+
+If you just want to look up the latest value for a particular key (let's assume `k2` in this case), then you can simply issue a pull query for that particular key:
+```sql
+SELECT * FROM latest WHERE k = 'k2';
++------+------+------+------+
+|K     |V1    |V2    |V3    |
++------+------+------+------+
+|k2    |4     |e     |true  |
+```
+
+
+## Materializing a changelog `STREAM`
+
+The best way to materialize an input topic as a table is the `CREATE SOURCE TABLE` statement. If you already have a `STREAM`, though, you can also materialize it as a table with the `CREATE TABLE ... AS ...` statement. Let's say you have a `STREAM` of events in ksqlDB that represent a series of changes called `changelog_stream`, and you want a view of the data that reflects the latest values for each key. 
+
+Begin by telling ksqlDB to start all queries from the earliest point in each topic:
+```sql
+SET 'auto.offset.reset' = 'earliest';
+```
 
+Let's mimic adding a few records to the `changelog_stream` by using the `INSERT INTO` statement:
 ```sql
-INSERT INTO s1 (
+INSERT INTO changelog_stream (
     k, v1, v2, v3
 ) VALUES (
     'k1', 0, 'a', true
 );
 
-INSERT INTO s1 (
+INSERT INTO changelog_stream (
     k, v1, v2, v3
 ) VALUES (
     'k2', 1, 'b', false
 );
 
-INSERT INTO s1 (
+INSERT INTO changelog_stream (
     k, v1, v2, v3
 ) VALUES (
     'k1', 2, 'c', false
 );
 
-INSERT INTO s1 (
+INSERT INTO changelog_stream (
     k, v1, v2, v3
 ) VALUES (
     'k3', 3, 'd', true
 );
 
-INSERT INTO s1 (
+INSERT INTO changelog_stream (
     k, v1, v2, v3
 ) VALUES (
     'k2', 4, 'e', true
 );
 ```
 
-Derive a table, `t1`, from stream `s1`. When you create a table, the columns in the `SELECT` clause must either be the column that you're grouping by or columns with an aggregation function applied. The `LATEST_BY_OFFSET` aggregation allows you to select any column and retains only the last value it receives, where "last" is in terms of offsets.
+You can view the data in the `changelog_stream` by issuing a pull query:
+```sql
+SELECT * FROM changelog_stream;
++------+------+------+------+
+|K     |V1    |V2    |V3    |
++------+------+------+------+
+|k1    |0     |a     |true  |
+|k2    |1     |b     |false |
+|k1    |2     |c     |false |
+|k3    |3     |d     |true  |
+|k2    |4     |e     |true  |
+Query terminated
```

Comment:
Do we really print `Query terminated`? Why? For a push query it might make sense, but not for a pull query?